azure/stages.yml

```diff
@@ -58,6 +59,7 @@ stages:
          single_threaded: ${{ parameters.single_threaded }}
          nightly_feature: ${{ parameters.nightly_feature }}
          features: ${{ parameters.test_features }}
+         raw_opts: ${{ parameters.test_args_raw }}
```

Comment:
I think I'd like to see slightly different names for these. Specifically, let's use `test_arguments` in `stages.yml` and `arguments` in `test` and `tests`.